### PR TITLE
Bosnian translation

### PR DESCRIPTION
**Pull request by ivana-mcconnell**
I know that there are likely some grammatical errors in this as I haven’t written Bosnian in _years_, but I do think that it’s very useful as an opening salvo. Any edits and/or refinements are very welcome!

### CODE REVIEW

**Comment by CoralineAda** [comment]
I've put out a call for native speakers to review. Thanks for starting this @ivana-mcconnell !

**Comment by ivana-mcconnell** [comment]
No worries! I'm a native speaker but not a native writer (if that makes sense), so I'd love to see how this evolves. 🙂

File: version/1/4/bs/code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Saradnik Pakt: Kodeks Ponašanja
+
+## Naš zalog
+
+U interesu ohrabrujući otvoren i dobrodošlice okruženje,mi kao saradnici i održavatelji obećamo da za sviju ko učestvuju u našem projektu i zajednicu, njihovo iskustvo  će biti bez uznemiravanje, bez obzira na starost, veličinu, invalidnost, nacionalnost, rodnog identiteta i izraz, nivo iskustva, državljanstvo, izgled, porod, vjera, ili seksualni identitet i orijentacija.
```
Comment (mislav) [inline version/1/4/bs/code_of_conduct.md]: * U interesu ohrabr**ivanja** otvoren**og** i pristupačnog okruženja ("dobrodošlog" ne funkcionira dobro kao pridjev ovdje)
* obeća**vamo**
* “da za sviju ko učestvuju u našem projektu i zajednicu, njihovo iskustvo  će biti bez uznemiravanje” - ovo mi ne zvuči dobro, jer dosta padeža ne štima, ali za sada nemam bolji prijedlog
* rodni identitet
* orijentacij**e***

File: version/1/4/bs/code_of_conduct.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Naši standardi
+
+Primjeri ponašanja što stvori pozitivno okruženje:
```
Comment (mislav) [inline version/1/4/bs/code_of_conduct.md]: “stvara” je možda bolje?

File: version/1/4/bs/code_of_conduct.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+* Korištenje prijatan i uključiv jezik
```
Comment (mislav) [inline version/1/4/bs/code_of_conduct.md]: prijat**nog** i uključiv**og**

**Review by mislav — COMMENTED** [review]
Hi Ivana! I'm not Bosnian but I'm Croatian, and that's close. Thank you for your contribution! I have started reviewing the text, but it seems to me that there needs to be a lot more work done on this before it's ready, as there seem to be a lot of errors with cases. I would like to help rewrite some of the more complex sentences, but I fear that I don't know very well the subtle boundaries of our languages and that I might express myself in Croatian in places where Bosnian is different. So, I suggest that we find someone who is really proficient with the Bosnian language who can make a pass at this. How does that sound?

In Croatian: Zdravo! Meni se čini da je dosta padeža pogrešno u ovom tekstu. Znam da bosanski nije identičan hrvatskom, ali mislim da su nam padeži jednaki. Mislim da trebamo pomoć od nekoga tko trenutno priča bosanski da pregleda ovaj tekst 😄 Znaš li nekoga od tvojih prijatelja/rodbine?

**Comment by ivana-mcconnell** [comment]
Hi @mislav! Thanks so much for having a look! I'd expected some case and conjugation errors in it, particularly for some of the more complex sentences!

> I don't know very well the subtle boundaries of our languages and that I might express myself in Croatian in places where Bosnian is different.

That's interesting— my family is Serbian, Croatian, and Bosnian, though I grew up speaking the latter. I imagine some of my errors stem from that as well :) Finding someone who writes Bosnian often and fluently to have a look over it would be great! Know anyone?

File: version/1/4/bs/code_of_conduct.md (hunk 1 above)
Comment (vvuksan) [inline version/1/4/bs/code_of_conduct.md]: This is also Croatian but this is my attempt at rewritting that passage

U interesu ohrabrivanja otvorenog i pristupačnog okruženja, mi kao saradnici i odrzavatelji obecavamo svima koji ucestvuju u nasem projektu i zajednici da ce njihovo iskustvo biti bez uznemiravanja, bez diskriminacije na osnovu dobi, invalidnosti, nacionalnosti, rodnog identiteta i izraza, nivoa iskustva, drzavljanstva, izgleda, poroda, vjere ili sekusalnog identiteta ili orijentacije.

File: version/1/4/bs/code_of_conduct.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+* Poštovanje drugačije stanovište i iskustve
```
Comment (vvuksan) [inline version/1/4/bs/code_of_conduct.md]: Postovanje drugacijeg stanovista i iskustva

File: version/1/4/bs/code_of_conduct.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+* Primanje konstruktivne kritike
+* Fokusiranje na šta je najbolje za zajednicu
+* Pokazivanje empatiju za druge članovi zajednice
```
Comment (vvuksan) [inline version/1/4/bs/code_of_conduct.md]: Pokazivanje empatije drugim clanovima zajednice

File: version/1/4/bs/code_of_conduct.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+Primjeri neprihvatljivog ponašanja:
+
+* Korištenje seksualnog jezik ili neželjene seksualne pažnje
```
Comment (vvuksan) [inline version/1/4/bs/code_of_conduct.md]: jezik**a**

File: version/1/4/bs/code_of_conduct.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+* Uvredljivi komentari ili personalni napadi
+* Javno ili privatno uznemiravanje
+* Izdavanje privatnu informaciju, na primjer adresu, bez dozvole
```
Comment (vvuksan) [inline version/1/4/bs/code_of_conduct.md]: Objava osobnih informacija, npr. kucna adresa bez prethodne dozvole

File: version/1/4/bs/code_of_conduct.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+* Bilo kakvo drugo ponašanje što bi bilo neprikladno u profesionalnom okruženju.
```
Comment (vvuksan) [inline version/1/4/bs/code_of_conduct.md]: Bilo kakvo drugo ponasanje koje bi se smatralo neprikladnim u profesionalnom okruzenju.

File: version/1/4/bs/code_of_conduct.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+## Naši odgovornosti
```
Comment (vvuksan) [inline version/1/4/bs/code_of_conduct.md]: Nasa odgovornost

File: version/1/4/bs/code_of_conduct.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+Održavatelji projekta će razjasniti standarde za prihvatljivo ponašanje, i u slučaju neprikladno ponašanje, od njih se očekuje poštene korektivne mjere.
```
Comment (vvuksan) [inline version/1/4/bs/code_of_conduct.md]: i u slucaju neprikladno**g** ponasanj**a** od njih se ocekuje primena postenih korektivne mjere

File: version/1/4/bs/code_of_conduct.md (hunk 2 above)
Comment (triggerNZ) [inline version/1/4/bs/code_of_conduct.md]: "koja stvaraju"?

File: version/1/4/bs/code_of_conduct.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+
+Održavatelji projekta imaju pravo i odgovornost da skinu, redigovatu, ili odbitu komente, kod, wiki izmjene, probleme, i druge doprine koji se ne poređaju sa ovom Kodeks Ponašanja. Oni imaju pravu da zabrane (prolazno ili trajno) bilo koji saradnik za ponašanje koje se pojavljuje neprijatno, štetno, ili uvredljivo.
+
+## Obim
+
+Ovaj kodeks ponašanja se odnosi na sve projekte i javne prostore gdje pojedinac predstavlja zajednicu. Na primjer, korištenje od službenu e-mail adresu projekta ili ponašanje kao sluzbeni predstavnik u sastancima. Održavatelji projekta će dodatno definirati ovu predstavljanju.
```
Comment (triggerNZ) [inline version/1/4/bs/code_of_conduct.md]: korištenje službene e-mail adrese

... ovo predstavljanje

File: version/1/4/bs/code_of_conduct.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+
+## Primjenjivanje
+
+U slučaju uvredljivog ponašanja, kontaktiraj održavatelje: [E-MAIL ADRESA]
```
Comment (triggerNZ) [inline version/1/4/bs/code_of_conduct.md]: kontaktirajte

**Review by triggerNZ — CHANGES_REQUESTED** [review]
Croatian speaker here, fairly polluted by years of English so take my comments with a grain of salt, but I do have relatives in Bosnia so am familiar with certain Bosnian words idioms, though not all of them

**Comment by triggerNZ** [comment]
@ivana-mcconnell how would you feel about me forking this and doing a HR version?

File: version/1/4/bs/code_of_conduct.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+
+Održavatelji projekta će razjasniti standarde za prihvatljivo ponašanje, i u slučaju neprikladno ponašanje, od njih se očekuje poštene korektivne mjere.
+
+Održavatelji projekta imaju pravo i odgovornost da skinu, redigovatu, ili odbitu komente, kod, wiki izmjene, probleme, i druge doprine koji se ne poređaju sa ovom Kodeks Ponašanja. Oni imaju pravu da zabrane (prolazno ili trajno) bilo koji saradnik za ponašanje koje se pojavljuje neprijatno, štetno, ili uvredljivo.
+
+## Obim
+
+Ovaj kodeks ponašanja se odnosi na sve projekte i javne prostore gdje pojedinac predstavlja zajednicu. Na primjer, korištenje od službenu e-mail adresu projekta ili ponašanje kao sluzbeni predstavnik u sastancima. Održavatelji projekta će dodatno definirati ovu predstavljanju.
+
+## Primjenjivanje
```
Comment (triggerNZ) [inline version/1/4/bs/code_of_conduct.md]: Primjena ?

**Comment by mislav** [comment]
Hvala @vvuksan @triggerNZ na pomoći!

@triggerNZ I'm only speaking for myself here, but I think you'd definitely be welcome to make a Croatian translation based on this Bosnian version. I think the best time to start your HR translation would be when this Bosnian one gets finalized and merged, so there would be less work for you keeping both sides in sync when they are developing in parallel.

**Comment by ivana-mcconnell** [comment]
@triggerNZ Go for it! 

@mislav — once I make the changes suggested by yourself and @vvuksan, do you think it will be ready, or will it need more review?

**Comment by mislav** [comment]
@ivana-mcconnell Thanks for making all those updates! Please ping the thread after you're done and I think that it will be enough that at least one of us giving our 👍 should be enough for this to be considered mergeable.

File: version/1/4/bs/code_of_conduct.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+
+U slučaju uvredljivog ponašanja, kontaktiraj održavatelje: [E-MAIL ADRESA]
+
+Svaki izvještaj će se istražiti, i odgovor će biti fer i prikladan za situaciju. Održavatelji projekta su odgovorni da održavaju povjerljivost za svaki izvještaj. Dodatne detalje polisa mogu biti objavljeno odvojeno.
```
Comment (vvuksan) [inline version/1/4/bs/code_of_conduct.md]: Mislim da je izvjestaj pretjerano kruto. Mislim da je bolje reci zalba (poput knjige zalbi) znaci

Svaka zalba ce se istraziti i odgovor ce biti fer i prikladan za situaciju. Odrzavatelji projekta su odgovorni odrzati povjerljivost svake zalbe.

**Comment by ivana-mcconnell** [comment]
@mislav @vvuksan @triggerNZ I've made the changes, let me know if I've missed anything!

**Comment by ivana-mcconnell** [comment]
@CoralineAda this might be ready to merge now? :)

**Comment by CoralineAda** [comment]
Thanks for your hard work!

**Review by KunKax — COMMENTED** [review]
!

**Review by Alex43ice — COMMENTED** [review]
- []()google_privacy_policy_en_us.pdf

**Review by Alex43ice — COMMENTED** [review]
Thanks 👍

**Review by Alex43ice — COMMENTED** [review]
[]()https://github.com/orgs/community/discussions

**Review by Alex43ice — COMMENTED** [review]
Awesome

**Comment by Alex43ice** [comment]
Great

File: version/1/4/bs/code_of_conduct.md (hunk 14 above)
Comment (DEN19875) [reply version/1/4/bs/code_of_conduct.md]: README.md

_

Comment (DEN19875) [reply version/1/4/bs/code_of_conduct.md]: -```
- - [ ] # > `~~@Резервные коды.pdf

 ~~`
```

Comment (DEN19875) [reply version/1/4/bs/code_of_conduct.md]: > README.md
> 
> _

Comment (DEN19875) [reply version/1/4/bs/code_of_conduct.md]: [](

**Pull request merged**